lua/ttt2/extensions/cvars.lua

```diff
@@ -0,0 +1,174 @@
+---
+-- cvars extensions
+-- @author ZenBreaker
+-- @module cvars
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+local cvars = cvars
+
+local messageIdentifier = -1
+local identityBitCount = 8
+local maxUInt = 2 ^ identityBitCount
+
+local serverConVars = {}
+local functionCache = {}
```

Comment:
```suggestion
local functionCache = functionCache or {}
```

I guess this might otherwise be cleared on hot reload, but I'm not 100% sure.

Reply:
yeah true. Same with serverConVars

Reply:
If this is local has hotreloading any effect on it? Otherwise it stated that the local cache is either the global functionCache or empty table. I guess, this would be an error 🤔

Reply:
if it is local, it can't be persistent while hotreloading